### ISSUE TITLE
demojail.dis.ulpgc.es is down

### ISSUE THREAD

**Issue by alefortvi** [jcrodriguez-dis/vpl-jail-system#78]
I'm trying to use the Moodle's VPL in a classroom towards the server demojail.dis.ulpgc.es and I see it's down. Could be?

**Comment by mxmerz** [comment]
I believe the idea of the demojail server is that you can use it for testing purposes (to evaluate whether VPL is good for your usecase), but not for actual classroom use… (See https://vpl.dis.ulpgc.es/documentation/vpl-3.4.3+/installation.html#execution-servers-list)

I installed the VPL Jail System (the software in this repo) with the [installation instructions](https://vpl.dis.ulpgc.es/documentation/vpl-jail-system-3.0.1/installation.html) on a Raspberry Pi 4 with Ubuntu a few days ago, that worked very well.

**Comment by alefortvi** [comment]
That's correct! I'm a teacher and I'm setting up a space with VPL, and I had configured the server demojail.dis.ulpgc.es to test the Programming Laboratory tool. If everything goes well, I'll ask the support team to set up our own server.

**Comment by jcrodriguez-dis** [comment]
Dear @alefortvi,

Thank you for reaching out about your experience with the demojail server. I apologize for the inconvenience; indeed, you've encountered some downtime this week due to our ongoing server updates. This isn't typical and I appreciate your understanding.

Please feel free to continue using the server for your testing purposes, and let us know if you encounter any further issues.

Best regards,

**Comment by vilelaricardo** [comment]
Dear @jcrodriguez-dis ,

I'm not having success accessing the server. Has the `demojail.dis.ulpgc.es` server been disabled?

Regards,
Ricardo Vilela

**Comment by jcrodriguez-dis** [comment]
Dear @vilelaricardo,

We did not experience any downtime on the demojail server on May 16 and 17. The issue might be on your side. Additionally, please ensure you are using HTTPS instead of HTTP.

Best regards,
Juan Carlos

**Comment by vilelaricardo** [comment]
Thank you, @jcrodriguez-dis!

I was getting the following response when trying to perform an action in VPL:
```json
{
"success": false,
"response": {},
"error": "Argument 1 passed to vpl_jailserver_manager::get_https_server_list() must be of the type string, null given, called in /var/www/html/moodle-4.1/mod/vpl/forms/edit.json.php on line 132"
}
```

After seeing your comment to clear the server list in this post: [[link](https://moodle.org/mod/forum/discuss.php?d=423304)], I managed to solve the issue.

Thank you.

Regards
